GitHub_LearningTests/Generic/Utilities/Infrastructure/StringExtensionTests.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  StringExtensionTests.swift
+//  GitHub_LearningTests
+//
+//  Created by Amin Ashhab on 16.12.2020.
+//  Copyright © 2020 Amin_Second_Test_Project. All rights reserved.
+//
+
+import XCTest
+@testable import GitHub_Learning
+
+class StringExtensionTests: XCTestCase {
+    private var sut: String!
+    
+    func setUpTestWith(string: String) {
+        sut = string
+    }
+    
+    func test_givenNumberInString_thenStringContainsNumber() {
+        // Given
+        setUpTestWith(string: "1")
+        
+        // Then
+        XCTAssertTrue(sut.containsNumber)
```

Comment:
Tydle testy by si zaslouzily aby v jednom testu bylo klidne vice assertu. Tj min test funkci a bylo by to prehlednejsi. Neco takoveho: Budes mit pole stringu a booleanu (tuple), ve kterem bude jestli obsahuje cislo nebo ne. Pak to pole cely projdes a jen zkontrolujes ze ta `.containsNumber` proprety je stejna jako tvuj bool. Stejne tak u ostatnich properties.